portal/templates/reset-password.njk

```diff
@@ -19,19 +21,48 @@
       <form method="POST" autocomplete="off">
         <input type="hidden" name="_csrf" value="{{ csrfToken }}">
         <div class="govuk-form-group govuk-!-width-one-third">
-          <label class="govuk-label" for="email">
-            Email
-          </label>
-          <input data-cy="email" autocapitalize="off" class="govuk-input" id="email" name="email" type="text">
+           {{ govukInput({
+            label: {
+              text: "Email address"
+            },
+            id: "email",
+            name: "email",
+            spellcheck: false,
+            type: "email",
+            autocomplete: "email",
+            errorMessage: errors.fieldErrors.email and {
+              text: errors.fieldErrors.email.text,
+              attributes: {
+                "data-cy": "reset-password-email-error"
+              }
+            },
+            attributes: {
+              "data-cy": "reset-password-email",
```

Comment:
the previous `data-cy` was set to `email`. do we need to change that anywhere else?

Reply:
Nope.